src/test/a11y/tests/system-admin-routes.test.ts

```diff
@@ -125,4 +125,46 @@ describe('Accessibility - System Admin Routes', () => {
             testAccessibility(route.path, route.parameter);
         });
     });
+
+    describe('Page with Errors', () => {
+        describe('Create System Admin Account Page', () => {
+            const url = '/create-system-admin-account';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
```

Comment:
'input-autocomplete' is not the right post body for this test. Please check the controller code